### PR TITLE
Retrieve subtitles from youtube instead of amara

### PR DESCRIPTION
**Pull request by mrpau-eduard**
## Summary
Since Khan Academy is no longer using amara for subtitling we will download the subtitles form `YouTube` and used youtube-dl to download the subtitles.  

## TODO

- [x] Rebuild all the language packs for testing
- [x] Test by @radinamatic 

## Issues addressed
#66

### CODE REVIEW

**Comment by radinamatic** [comment]
@mrpau-eduard Ping me when the contentpacks are ready!

File: contentpacks/khanacademy.py [hunk 1]
```diff
@@ -147,57 +150,32 @@ def retrieve_language_resources(version: str, sublangargs: dict, ka_domain: str,
...
-                amara_id, lang)
-            filename = "subtitles/{lang}/{youtube_id}.vtt".format(lang=lang, youtube_id=youtube_id)
-            subtitle_path = download_and_cache_file(subtitle_download_uri, filename=filename, ignorecache=False)
-            logging.info("subtitle path: {}".format(subtitle_path))
+        # Store the file temporarily, then rename it later since it has a wrong extension used 
+        # e.g {youtube_id}. {Lang}. vtt so we need to rename it.
+        dump_file = "%s/%s/%s" % (SUBTITLE_DIR, lang, youtube_id)
+        download_cmd = "youtube-dl --sub-format 'vtt' --sub-lang '{lang}' --write-srt -o '{path}'  --skip-download 'https://www.youtube.com/watch?v={youtube_id}'".format(
+            path=dump_file, lang=lang, youtube_id=youtube_id)
+        os.system(download_cmd)
```
Comment (benjaoming) [inline contentpacks/khanacademy.py]: Not that important, but just so you know: Youtube-dl is actually a Python library so can be accessed directory: https://github.com/rg3/youtube-dl/blob/master/README.md#embedding-youtube-dl

File: requirements_dev.txt [hunk 2]
```diff
@@ -11,4 +11,5 @@ wrapt==1.10.5
 pyflakes==1.0.0
 pytest==2.8.2
 pytest-asyncio==0.2.0
+youtube-dl==2017.06.12
```
Comment (benjaoming) [inline requirements_dev.txt]: This isn't a dev requirement? Should be in `requirements.txt`...

**Review by benjaoming — CHANGES_REQUESTED** [review]

**Comment by mrpau-eduard** [comment]
Hi @radinamatic, you can grab the language packs here
https://drive.google.com/open?id=0B-E59G3J_sXmazlqUHljaDVTcTQ

**Comment by mrpau-eduard** [comment]
HI @benjaoming I will revise the code base on your comments thanks for reviewing :)

**Comment by radinamatic** [comment]
@mrpau-eduard I installed the latest contentpacks, and downloaded several random videos in English, and they all correctly present the subtitles, so since I cannot actually test all 8k+, I'm giving this PR a pass! :wink: 

If we discover that some are missing, we'll take it on case-by-case basis.

On another hand (I'll separate this in a new issue if necessary), I'm still seeing (a totally useless) English subtitles on dubbed Spanish videos... :thinking:   

xubuntu start running - oracle vm virtualbox_355

**Comment by mrpau-eugene** [comment]
@radinamatic I think you are probably using the `0.17.1` release. We disabled it in `0.17.2`

**Comment by radinamatic** [comment]
Ok, I will re-test with the latest 0.17.2 installer, and report back! :+1:

**Comment by mrpau-eduard** [comment]
Hi @benjaoming can you review it again? I update my `PR` base on your comments.

**Review by benjaoming — APPROVED** [review]

**Comment by benjaoming** [comment]
Looks great! Ready to merge and start building content packs, no?

**Review by radinamatic — APPROVED** [review]
LGTM

**Comment by radinamatic** [comment]
Re-tested with `0.17.2-rc2` installer, good to merge, but we need to fix #71 before building contentpacks again (partners in south America are expecting this).

**Comment by mrpau-eduard** [comment]
@radinamatic Yes, we should address the #71 issue. We will take a look on it.

**Comment by mrpau-eduard** [comment]
Hi @benjaoming I made an update on my script. We can merge it now.
I build a new `sw` to test the updates and it is working.

**Comment by benjaoming** [comment]
@cpauya responded to your comment - IMO this PR looks good to go :tada:

**Pull request merged**